api/zeebe/generation-strategy.js

```diff
@@ -15,6 +17,36 @@ module.exports = {
   postGenerateDocs,
 };
 
+function hackChangesetDescription() {
+  // This is a temporary hack, until https://github.com/camunda/camunda-docs/issues/3568 is resolved.
+  //   The OpenAPI generator plugin we're using does not use the correct `description` property
+  //   for the `UserTaskUpdateRequest` object. Instead of picking up the actual property description,
+  //   it picks up the description of the first merged schema in the `allOf` property (i.e. from the `Changeset` schema).
+  // This adjustment replaces the description of the `Changeset` schema with the current description of
+  //   the `UserTaskUpdateRequest.changeset` property.
+  console.log("hacking changeset description...");
+  replace.sync({
+    files: `${specFile}`,
+    from: /^      description: A map of changes.$/m,
+    to: `      description: |
+        JSON object with changed task attribute values.
+
+        The following attributes can be adjusted with this endpoint, additional attributes
+        will be ignored:
+
+        * \`candidateGroups\` - reset by providing an empty list
+        * \`candidateUsers\` - reset by providing an empty list
+        * \`dueDate\` - reset by providing an empty String
+        * \`followUpDate\` - reset by providing an empty String
+
+        Providing any of those attributes with a \`null\` value or omitting it preserves
+        the persisted attribute's value.
+
+        The assignee cannot be adjusted with this endpoint, use the Assign task endpoint.
+        This ensures correct event emission for assignee changes.`,
+  });
+}
+
```

Comment:
❓ Do we still need this when we also adjusted the YAML manually?
💭 We could add a comment to the YAML here in the docs project that this is manually adjusted currently. In the future, when we replace the YAML, we can see this change popping up. We could remove this hacky workaround then.

Reply:
My goal is to make the generation process as mistake-proof as possible. We _can_ rely on humans to catch the diff when they re-generate the docs and revert it, but it's nearly as likely that the diff would be overlooked, no one catches it in review, and the description is reverted in production.

The workflow I'm envisioning is that an alpha ships, someone grabs the updated OpenAPI spec, pastes it into `zeebe-openapi.yaml` here, and then runs `npm run api:generate zeebe`.... _then_ looks at the diff to see what's changed, or just opens a PR without looking at the diff. 
 
This step is written in a way that it will only replace the description if it still has the "incorrect" value. If the person re-generating the docs notices the diff _before_ re-generating, and reverts the description change, this action is effectively a noop. 

I don't feel good about hardcoding the contents of the description in this hack, but I do feel confident that this hack will prevent round-trips during re-generation, where the reverted description is not noticed until the PR is open. If you look at [the tasklist pre-generation script](https://github.com/camunda/camunda-docs/blob/4e82bbfeeaea7174bb1fec0b636fd7267a8d1ebc/api/tasklist/generation-strategy.js#L9-L10), or [the operate pre-generation script](https://github.com/camunda/camunda-docs/blob/main/api/operate/generation-strategy.js#L8), you can see other types of changes that can also be caught in this way. Eventually I hope to get all of those issues cleaned up at the source, but right now it's a nice way for me to automate cleanup that I'm doing upon generation, and I see this description as similar.

Reply:
OK, I see, good arguments. Let's roll with that 👍